### PR TITLE
[Android] Fix for Bullet Overlay

### PR DESCRIPTION
**Pull request by molecula451**
So far we're still having fun with our style elements it was the time for our cursor-overlay element <code> .android .bullet-cursor-overlay </code> since it is our state class for our Android elements.

Even though our properties are very well rendering our bullets and positioning. The <code> overlay</code> itself was not measure according to our needs.

### Actual issue: #211 

70553055-d5694100-1b50-11ea-8a90-95e4b91f18d1

### Expected Behaviour:

overlay

- [x] Bullet diameter should be about equal to half the height of a capital letter, as shown.
- [x] Bullet overlay diameter should be slightly less than the full height of a capital letter, as shown.
- [x] The spacing between the bullet and the text may need to adjusted to compensate for the larger bullet size. The relative spacing can be seen in the reference screenshot.
- [x] These requirements must be made for all six bullet types.

### CODE REVIEW

**Comment by molecula451** [comment]
@raineorshine How it's this one looking for you?

**Comment by raineorshine** [comment]
Thanks! Can you reference the issue number, please?

> So far we're still having fun with our style elements it was the time for our cursor-overlay element .android .bullet-cursor-overlay since it is our state class for our Android elements.

> Even tough our properties are very well rendering our bullets and positioning the overlay itself was not measure according to our needs.

I'm having trouble understanding your English here. Can you try to clarify?

**Comment by raineorshine** [comment]
Please send screenshots when you believe all requirements are complete. Do not send screenshots for incomplete work unless you are blocked and have questions about how to move on. Thanks!

**Comment by molecula451** [comment]
> Thanks! Can you reference the issue number, please?
> 
> > So far we're still having fun with our style elements it was the time for our cursor-overlay element .android .bullet-cursor-overlay since it is our state class for our Android elements.
> 
> > Even tough our properties are very well rendering our bullets and positioning the overlay itself was not measure according to our needs.
> 
> I'm having trouble understanding your English here. Can you try to clarify?

Even though* , tiny mistypo. Issue already referenced

File: src/App.css [hunk 1]
```diff
@@ -753,34 +758,50 @@ h1 .num-contexts {
...
 }
 
+/* overlay bullet for drag */
+/* specificity to beat ".dark .child.editing > .thought .bullet-cursor-overlay" */
+.android .child.child.child.dragging > .thought .bullet-cursor-overlay {
+  color: lightblue;
+  opacity: 1;
+}
+
+/* Our Awesome first child (overlay) independent with its own properties */
```
Comment (molecula451) [inline src/App.css]: Some new comments added for code maintenance.

**Comment by molecula451** [comment]
- Added .android classes for <code> .bullet .glyph </code> and <code> .bullet-cursor-overlay   </code > and fixed.

Hi Raine. Hope you are doing great. I took a chance on this one to re-factor and applied some new properties for the enhacement.

The overlay so far it's now rendering as expected in both. Portrait (Small phones) Aswell as on tablet.

We have: 

**Arrow Down context view activated:**

2019_12_19_11 49 31_edit

**Closed Arrow Down**
2019_12_19_11 48 37_edit

**Open Circle Context View** activated (I am a tiny doubtful about this one but IMO it is an pretty well looking this way)

2019_12_19_11 47 58_edit

**Closed Circle** (This snap it's somehow extra zoomed )
2019_12_19_11 47 32_edit

We can't see (Overlays) when our bullets are not touched 

- Open Arrow Right
- Open Circle (Not touching)
- Closed Circle (Not touching

But the alignment still on a well good fit.

2019_12_19_11 50 26_edit

**Comment by raineorshine** [comment]
This looks great! Thanks for your persistence and careful attention to detail.

Please resolve the merge conflicts and then I will accept.

**Comment by molecula451** [comment]
> This looks great! Thanks for your persistence and careful attention to detail.
> 
> Please resolve the merge conflicts and then I will accept.

I believe it's pretty much done Raine.

**Pull request merged**